webhook-bridge/svix-webhook-bridge.example3.yaml

```
@@ -0,0 +1,74 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on when receiving webhooks
+receiver_http_addr: "0.0.0.0:5000"
```

```suggestion
listen_address: "0.0.0.0:5000"
```
1. I'd keep it generic because I think we'll maybe use it for sender too at some point (as mentioned before).
2. No need to abbreviate, doesn't add anything.